mesa/experimental/cell_space/voronoi.py

```diff
@@ -0,0 +1,58 @@
+from collections.abc import Sequence
+from itertools import combinations
+from random import Random
+from typing import Optional
+
+from pyhull.delaunay import DelaunayTri
```

Comment:
I'm a bit worried about adding this package as a dependency. It's last update was in 2015.

Maybe we can use SciPy?
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.Delaunay.html
- https://docs.scipy.org/doc/scipy/tutorial/spatial.html#delaunay-triangulations

Otherwise maybe `shapely.ops.triangulate` or keeping the code in our own repo.

Reply:
Hmmm, I choose PyHull because Mesa is not using Scipy already and it is a very heavy dependency. I'll take a look at Shapely, it looks a good alternative, and as you said, can be integrated with Mesa Geo

Reply:
Sounds reasonable!